src/screens/SignIn/form.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import FormButton from 'components/FormButton'
+
+type SignInFormProps = {
+  submitHandler?: React.FormEventHandler
+}
+
+const SignInForm = ({ submitHandler }: SignInFormProps): JSX.Element => {
+  return (
+    <form id="signInForm" onSubmit={submitHandler}>
```

Comment:
Can we add some frontend validation in the form as HTML validation is not really helpful! 🤔

Reply:
What kind of validation? and could you please explain more a bit why the HTML validation is not helpful?
Thank you

Reply:
@gutakk simple `required` validation with JS should be enough. 

HTML validations are easily bypassed by the inspecting feature of browsers. So it's better to have a check with JS. 😆

Reply:
That make sesnse. Added in 73419e810fdc20a68a4c8c171a2c1551014fc68c